raster/r.texture/testsuite/test_texture.py

```diff
@@ -24,59 +25,164 @@ def setUpClass(cls):
     def tearDownClass(cls):
         cls.del_temp_region()
         cls.runModule("g.remove", flags="f", type="raster", name="asm_ASM")
+        cls.runModule("g.remove", flags="f", type="raster", name="contrast_Contr")
         cls.runModule("g.remove", flags="f", type="raster", name="corr_Corr")
-        cls.runModule("g.remove", flags="f", type="raster", name="sa_SA")
         cls.runModule("g.remove", flags="f", type="raster", name="var_Var")
         cls.runModule("g.remove", flags="f", type="raster", name="idm_IDM")
+        cls.runModule("g.remove", flags="f", type="raster", name="var_Var")
+        cls.runModule("g.remove", flags="f", type="raster", name="sa_SA")
+        cls.runModule("g.remove", flags="f", type="raster", name="sv_SV")
+        cls.runModule("g.remove", flags="f", type="raster", name="se_SE")
+        cls.runModule("g.remove", flags="f", type="raster", name="entr_Entr")
+        cls.runModule("g.remove", flags="f", type="raster", name="dv_DV")
+        cls.runModule("g.remove", flags="f", type="raster", name="de_DE")
+        cls.runModule("g.remove", flags="f", type="raster", name="moc1_MOC-1")
+        cls.runModule("g.remove", flags="f", type="raster", name="moc2_MOC-2")
```

Comment:
The original code was not ideal, g.remove can take a list of rasters: name is defined as `name=string[,string,...]`, i.e., multiple values are allowed. (alternatively, you could just loop though them, but that's likely a bit slower)

Reply:
The duplication of names with the test methods is not idea, but I don't have a _good_ suggestion for that.

Reply:
Solved in the new commit.